doc/how_to/custom_components/reactive_html/reactive_html_layout.md

```diff
@@ -0,0 +1,292 @@
+)
+```
+
+Please note that you must
+
+- wrap the `{% for object in objects %}` loop in a HTML element with an `id`. Here it is wrapped
+with `<div id="container" class="pn-container">...</div>`.
+- close all HTML tags! `<hr>` is valid HTML, but not valid with `ReactiveHTML`. You must close it
+as `<hr/>`.
+
+Please note you can optionally
+
+- get the index of the `{% for object in objects %}` loop via `{{ loop.index0 }}`.
+
+## A list like layout
+
+If you want to create a *list like* layout similar to `Columnn` and `Row`, you can
+combine `NamedListLike` and `ReactiveHTML`.
+
+```{pyodide}
+import panel as pn
+import param
+
+pn.extension()
+
+class ListLikeLayout(pn.layout.base.NamedListLike, pn.reactive.ReactiveHTML):
+    objects = param.List()
+
+    _template = """
+    <style>
+    .pn-container {height: 100%;width: 100%;}
+    </style>
+    <div id="container" class="pn-container">
+         {% for object in objects %}
+            <h1>Object {{ loop.index0 }}</h1>
+            <div id="object">${object}</div>
+            <hr/>
+		{% endfor %}
+    </div>
+"""
+
+layout = ListLikeLayout(
+    "I love beat boxing",
+    "https://upload.wikimedia.org/wikipedia/commons/d/d3/Beatboxset1_pepouni.ogg",
+    "Yes I do!",
+    styles={"border": "2px solid lightgray"},
+)
+layout
+```
+
+Please note that you must
+
+- list `NamedListLike, ReactiveHTML` in exactly that order when you define the class! The other
+way around `ReactiveHTML, NamedListLike` will not work.
+
+You can now use `[...]` indexing and the `.append`, `.insert`, `pop`, ... methods that you would
+expect.
+
+## A layout of a dictionary
+
+If you want to layout a dictionary, you can use a for loop on the `.items()`.
+
+```{pyodide}
+layout[2]
+```
+
+import panel as pn
+import param
+
+pn.extension()
+
+class LayoutOfDict(pn.reactive.ReactiveHTML):
+    object = param.Dict()
+
+    _template = """
+    <style>
+    .pn-container {height: 100%;width: 100%;}
+    </style>
+    <div id="container" class="pn-container">
+         {% for key, value in object.items() %}
+            <h1>Item {{ loop.index0 }}</h1>
+            <div>{{ key }}</div>
+            <div id="value">${value}</div>
+            <hr/>
+		{% endfor %}
+    </div>
+"""
+
+LayoutOfDict(object={
+    "Intro":  "I **love** beat boxing",
+    "Example": "https://upload.wikimedia.org/wikipedia/commons/d/d3/Beatboxset1_pepouni.ogg",
+    "*Outro*": "Yes I do!"
+},
+    styles={"border": "2px solid lightgray"},
+)
+
+Please note
+
+- We can insert the `key` as a literal value using `{{ key }}`.
+- We must not give the HTML element containing `{{ key }}` an `id`. If we do, an exception will be
+raised.
+
+## A Grid like layout
+
+If you want to create a grid like layout similar to the [`GridSpec`](https://panel.holoviz.org/reference/layouts/GridSpec.html)
+you can ???
+
+Todo: Figure out how to do this. Looking at the `GridSpec` it seems there is no *mixin* class I can use.
```

Comment:
I've looked at `GridSpect` it does not seem like there is any easy to use *mixin* class to create a grid like layout.

Maybe I should just drop this example?

Reply:
I will drop this comment. There is enough to review already.